simon/_posts/2017-10-17-day-34.md

```diff
@@ -2,15 +2,20 @@
 title: Day 34
 date: 2017-10-17 19:00:00 -0000
 exercise: 0
-music: 0
+music: 20
 books: 0
 field_trip: 0
 ---
 Things to write about:
 
 * What did you study and work on today? Be sure to write down everything you did and the subjects (English, Math, Health, Social Studies, Science).
 * How many books did you finish since your last journal entry? **Record the number** and write what they were.
-* Did you do any music or crafts?
+* Did you do any music or crafts
 
-***
+**
 
+I wrote about creation and did 20 spelling words. Then I practiced some math and played some music then i wrote about Puntuations.
```

Comment:
"Creation" is a proper noun. It was a story you read. It'd be helpful to cite the Sioux, too.